src/ipyvizzu/templates/ipyvizzu.js

```diff
@@ -55,6 +55,25 @@ if (window.IpyVizzu?.version !== '__version__') {
 			this._moveHere(chartId, element)
 		}
 
+		plugin(element, chartId, plugin, options, name, enabled) {
+			this.charts[chartId] = this.charts[chartId].then((chart) => {
+				return import(plugin)
```

Comment:
Do we need to import the plugin each time? Could we store the plugin instance instead?